main/src/modules/Assembly.scala

```diff
@@ -29,6 +30,8 @@ object Assembly {
 
     case class Exclude(path: String) extends Rule
 
+    case class Relocate(from: String, to: String) extends Rule
```

Comment:
Later we can add more config on this to apply the shading only to specific libs.